pages/accounts.js

```diff
@@ -21,8 +22,11 @@ export default function Accounts() {
       <Head>
         <title>kidsFi - Finance for kids</title>
       </Head>
-      {user ? (
-        <p>Will follow</p>
+      <Layout />
+      {user && user.isChild ? (
```

Comment:
You could use optional chaining: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining